solr/core/src/java/org/apache/solr/core/PluginInfo.java

```diff
@@ -100,39 +100,23 @@ public String toString() {
     }
   }
 
+  /** From XML. */
```

Comment:
Maybe extend just a bit? From schema.xml? It made me look for a similar constructor that was /** From JSON. */ further down...

Reply:
This constructor supports XML config generally; not just a specific file as you wondered.  I added it because it may not be obvious to the reader who may be unfamiliar with `ConfigNode` but who would recognize XML `Node` type. The other constructors are more general; can't be said to be from any particular format.  I could drop this but was trying to be helpful.

Reply:
i think it's fine.   Thanks!